src/compiler/docs/readme/output-docs.ts

```diff
@@ -25,10 +43,20 @@ export const generateReadme = async (
   await Promise.all(
     readmeOutputs.map(async (readmeOutput) => {
       if (readmeOutput.dir) {
-        const readmeContent = generateMarkdown(userContent, docsData, cmps, readmeOutput);
-        const relPath = relative(config.srcDir, docsData.readmePath);
-        const absPath = join(readmeOutput.dir, relPath);
-        const results = await compilerCtx.fs.writeFile(absPath, readmeContent);
+        const relativeReadmePath = relative(config.srcDir, docsData.readmePath);
+        const readmeOutputPath = join(readmeOutput.dir, relativeReadmePath);
+
+        const currentReadmeContent =
+          readmeOutput.dir !== config.srcDir
```

Comment:
Double checking, only because the bug I fixed for the Lisbon team looked at lot like this - are these paths normalized if the user is:
- on Windows
- Just decided to write something like `C://Documents\ryan/why\ oh \why` in their README OT config

Reply:
I think there may be some circumstances where one or the other isn't normalized, so I'll just throw `normalizePath` on both to be safe